packages/apollo-language-server/src/project/client.ts

```diff
@@ -169,7 +169,8 @@ export class GraphQLClientProject extends GraphQLProject {
         total: totalTypes
       },
       tag: this.config.tag,
-      loaded: this.serviceID ? true : false,
+      loaded:
+        this.serviceID && (this.schema || this.serviceSchema) ? true : false,
```

Comment:
This checks to make sure the project actually _has_ a schema. Previously, this would pass if there are 0 types on both the client and service, and it would show users in the console that everything was fine. This is _not_ true :)

Reply:
Nit: `Boolean(this.serviceID && (this.schema || this.serviceSchema))`